lib/config/migration.ts

```diff
@@ -68,11 +68,6 @@ export function migrateConfig(
           migratedConfig[newKey] = true;
         }
         delete migratedConfig[key];
-      } else if (key === 'packageRules' && is.plainObject(val)) {
```

Comment:
🧐why do we have such a migration? 🤔🙈

Reply:
No idea, maybe some of migrations already can be removed

Reply:
I think I saw some people with a single packageRule mistakenly configured like this:

```json
{
  "packageRules": {
    "matchDatasources": ["maven"],
    "labels": ["maven"]
  }
}
```

We *could* drop this migration and give them an error instead.

Reply:
Migration removed